core/idl_types.h

```diff
@@ -25,5 +25,8 @@ struct IDLLongLong final : public IDLBaseHelper<int64_t> {};
 struct IDLLong final : public IDLBaseHelper<int32_t> {};
 struct IDLShort final : public IDLBaseHelper<int16_t> {};
 struct IDLString final : public IDLBaseHelper<std::string> {};
+// FIXME(Hwansung): should be generated automatically
```

Comment:
i think this lines should be generated automatically at another files.

Reply:
I think so too ! should be generated automatically ! and another files !